src/components/MiniMap.vue

```diff
@@ -90,4 +112,11 @@ $transition: 0.3s;
     transition-timing-function: ease-in, ease-in, linear;
   }
 }
+
+.mini-map-btn {
+  .btn-sm {
+    border-bottom-left-radius: 0;
+    border-top-left-radius: 0;
+  }
+}
```

Comment:
Since styles are only applied to one element, the scss nesting here isn't necessary and the selector could be `.mini-map-btn .btn-sm {`.